src/logger/BaseLogger_test.go

```diff
@@ -0,0 +1,225 @@
+package logger
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"strings"
+)
+
+func TestCreatesInstance(t *testing.T) {
+	logger := NewBaseLoggerAgent(MOCK_AGENT)
+	assert.NotNil(t, logger)
+	assert.Equal(t, logger.getAgent(), MOCK_AGENT)
+	assert.false(t, logger.getEnableable())
+	assert.false(t, logger.getEnabled())
+	assert.Nil(t, logger.getQueue())
+	assert.Nil(t, logger.getUrl)
+}
+
+func TestCreatesMultipleInstances(t *testing.T) {
+	agent1 := "agent1"
+	agent2 := "agent2"
+	agent3 := "agent3"
+	url1 := "https://resuface.io"
+	url2 := "https://whatever.com"
+	helper := NewTestHelper()
+	logger1 := NewBaseLoggerAgentUrl(agent1, url1)
+	logger2 := NewBaseLoggerAgentUrl(agent2, url2)
+	logger3 := NewBaseLoggerAgentUrl(agent3, helper.demoURL)
+
+	assert.Equal(t, logger1.getAgent(), agent1)
```

Comment:
should be `assert.Equal(t, agent1, logger1.Agent())` since `assert.Equal`'s arguments follow the pattern `(*testing.T, [expected val], [actual val])`